app/client/components/user/CreateUserForm.tsx

```diff
@@ -132,6 +135,8 @@ export default connect(state => {
   const selector = formValueSelector('createUser');
   return {
     businessUsageValue: selector(state, 'businessUsage'),
+    // @ts-ignore
+    organisation: state.organisation.data
```

Comment:
i thiiink we need to convert all the reducers to typescript to get the typing working correctly for this, would be really nice if we do

Reply:
yeah, it's annoying that it can't be incremental

Reply:
would be good to do at the same time as eliminating that YUGGGEEE amount of copypasta boilerplate